tests/report/interrupted/data/main.fmf

```diff
@@ -17,7 +17,20 @@
         test: exit 1
     /weird:
         order: 2
-        test: rm -f /bin/bash
+        test: |
+          case "$weird_method" in
+            bash)
+              rm -f /bin/bash
+              ;;
+            ssh)
+              pstree -p
```

Comment:
The only other place that I would question is this. Should we grep this to make sure sshd-srssion_pid is what it claims, or do we leave it for when it fails?

Reply:
By grep I meant `rlAssertGrep`